app/collins/util/parsers/LshwParser.scala

```diff
@@ -197,13 +197,29 @@ class LshwParser(txt: String) extends CommonParser[LshwRepresentation](txt) {
       val asset = getAsset(elem)
       // serial may be missing, so be flexible here and allow it to be absent
       val serial = (elem \ "serial" headOption).map(_.text)
-      ServerBase(asset.description, asset.product, asset.vendor, serial)
+      val motherboard = (elem \ "node" \ "product").headOption match {
```

Comment:
using Option[String] would simplify this boilerplate logic (/see line 199)

Reply:
Sounds good to me, changed in e64829b54711877d6342e15b8e9019a4b0440eb8.